modules/admin/proc-mounting-the-policy-csv-file-using-the-operator.adoc

```diff
@@ -0,0 +1,81 @@
+[id='proc-mounting-the-policy-csv-file-using-the-operator_{context}']
+= Mounting `policy.csv` file using the {product-short} Operator
+
+When the {product} is deployed with the Operator, you can add your `policy.csv` file using the {product-short} Operator by creating a `ConfigMap` and mounting it through your Custom Resource (CR).
+
+.Prerequisites
+
+* You are logged in to your {ocp-short} account using the {ocp-short} web console.
+* {product} is installed and deployed using the Operator.
+* You have added a custom configuration file to {ocp-short}. For more information, see link:{LinkAdminGuide}[Adding a custom configuration file to {ocp-short}].
++
+//For more information about installing the {product} on {ocp-short} using the Operator, see xref:proc-install-rhdh-ocp-operator_{context}[].
+//replace with a link to the installation guide.
```

Comment:
Should this be here?

Reply:
We already have a Jira for replacing the links, and another writer is currently addressing all the links in a single PR. Therefore, we can leave them as they are in this PR: https://issues.redhat.com/browse/RHIDP-3377.